src/hardware/mixer.cpp

```diff
@@ -587,6 +651,18 @@ spx_uint32_t estimate_max_out_frames(SpeexResamplerState *resampler_state,
 	return ceil_udivide(in_frames * ratio_den, ratio_num);
 }
 
+AudioFrame MixerChannel::ApplyCrossfeed(const AudioFrame &frame) const
+{
+	// Pan mono sample using -6dB linear pan law in the stereo field
+	// pan: 0.0 = left, 0.5 = center, 1.0 = right
+	auto pan = [](const float sample, const float pan) -> AudioFrame {
+		return {(1 - pan) * sample, pan * sample};
+	};
+	auto a = pan(frame.left, crossfeed.pan_left);
+	auto b = pan(frame.right, crossfeed.pan_right);
+	return {a.left + b.left, a.right + b.right};
```

Comment:
The use of lamda and AudioFrame really makes this function nice a elegant. Good stuff, @johnnovak :+1: